### ISSUE TITLE
From imports

### ISSUE THREAD

**Issue by EMattfolk** [sylt-lang/sylt-lang#353]
- [x] `from a use b` includes b from `a.sy`
- [x] `from a use b as c`  includes b from `a.sy` under name c
- [x] `from a use (b as c, d as e)`  includes b and d from `a.sy` under names c and e

**Comment by EMattfolk** [comment]
I suppose the easiest way of implementing this is to create the new variable and assign it to the value of the thing we are trying to import. It is kind of inconsistent to import it and then change it afterwards, so maybe only allow from-importing constants? Thoughts?

**Comment by FredTheDino** [comment]
You should only be allowed to import globals - since they're the variables that arn't limited to a scope. So you just need to copy the Name from one namespace to another - should be trivial to implement.

**Comment by EMattfolk** [comment]
What I mean is, should this be possible?
```
// a.sy
v := 0
f :: fn -> v end

// b.sy
from a use v, f

start :: fn do
    v += 1
    print(f()) // 1
end
```

**Comment by EMattfolk** [comment]
Closed in #565
